src/selftest-ro/configuration.json

```diff
@@ -1,32 +1,54 @@
-{
-	"usages": [
-		{
-			"usage_id": "discovery_monitoring",
-			"supported_types": [ "Snmpv3", "Snmpv1", "UserAndPassword" ]							
-		},
-		{
-			"usage_id": "mass_device_management",
-			"supported_types": [ "Snmpv3", "Snmpv1", "UserAndPassword" ]							
-		},
-		{
-			"usage_id": "nobody_access",
-			"supported_types": [ "Snmpv3", "Snmpv1", "UserAndPassword" ]							
-		}
-	],
-    "consumers": [
-		{
-			"client_regex": ".*",
-			"usages": [ "discovery_monitoring" ]							
-		},
-		{
-			"client_regex": "",
-			"usages": [ "nobody_access" ]							
-		}
-	],
-	"producers": [
-		{
-			"client_regex": ".*",
-			"usages": [ "discovery_monitoring", "mass_device_management" ]							
-		}
-	]
-}
\ No newline at end of file
+[
+	{
+		"portfolio_name": "default",
+		"usages": [
+			{
+				"usage_id": "discovery_monitoring",
+				"supported_types": [
+					"Snmpv3",
+					"Snmpv1",
+					"UserAndPassword"
+				]
+			},
+			{
+				"usage_id": "mass_device_management",
+				"supported_types": [
+					"Snmpv3",
```

Comment:
"mass device management" could be changed to "mass-management" for WR20.
For now, the mass management service never connect to device snmp servers.

Reply:
That's a file for unit test. No need to change. The run-time configuration file is going to be moved to BIOS/bios-image or some other image preparation scripts